src/Report/JunitReport.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Report;
+
+use Symfony\Component\Stopwatch\StopwatchEvent;
+
+class JunitReport extends AbstractReport
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(array $changed)
+    {
+        $fixEvent = $this->stopwatch->getEvent('fixFiles');
+        $time = sprintf('%.3f', $fixEvent->getDuration() / 1000);
+
+        /** @var StopwatchEvent[] $fixFiles */
+        $fixFiles = array_diff_key($this->stopwatch->getSectionEvents('fixFile'), array('__section__' => null));
+
+        $dom = new \DOMDocument('1.0', 'UTF-8');
+
+        $testsuites = $dom->appendChild($dom->createElement('testsuites'));
+
+        /** @var \DomElement $testsuite */
+        $testsuite = $testsuites->appendChild($dom->createElement('testsuite'));
+        $testsuite->setAttribute('name', 'PhpCsFixer'.($this->getApplication() ? '-'.$this->getApplication()->getVersion() : ''));
```

Comment:
:-1: for appending version to report.
especially if only that one will have it.